### PR TITLE
fix no-const-assign in Web/API

### PR DESCRIPTION
**Pull request by lionralfs**
#### Summary

Caught some assignments to `const` via [`no-const-assign`](https://eslint.org/docs/latest/rules/no-const-assign)

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/gamepad_api/using_the_gamepad_api/index.md [hunk 1]
```diff
@@ -270,7 +270,7 @@ function updateStatus() {
     scangamepads();
   }
 
-  const i = 0;
+  let i = 0;
   let j;
```
Comment (Josh-Cena) [inline files/en-us/web/api/gamepad_api/using_the_gamepad_api/index.md:274]: `i` and `j` should both be declared in the loop

Comment (lionralfs) [reply files/en-us/web/api/gamepad_api/using_the_gamepad_api/index.md:274]: I'll change it

Comment (lionralfs) [reply files/en-us/web/api/gamepad_api/using_the_gamepad_api/index.md:274]: We could probably use `for..of here` and get rid of the indices completely

**Review by teoli2003 — APPROVED** [review]
Thank you!

**Pull request merged**